LunaSysMgr.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Luna System Manager
-After=LunaSysService.service nyx.target
+After=LunaSysService.service nyx.target configurator.service
 Requires=LunaSysService.service
```

Comment:
@tofee @nizovn Not sure we need a requires here too? I guess not, but I'm not too familiar with systemd